central/sensor/service/pipeline/complianceoperator/complianceoperatorinfo/pipeline.go

```diff
@@ -32,10 +40,34 @@ func (s *pipelineImpl) Match(msg *central.MsgFromSensor) bool {
 }
 
 // Run runs the pipeline template on the input and returns the output.
-func (s *pipelineImpl) Run(_ context.Context, _ string, msg *central.MsgFromSensor, _ common.MessageInjector) error {
+func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.MsgFromSensor, _ common.MessageInjector) error {
 	defer countMetrics.IncrementResourceProcessedCounter(pipeline.ActionToOperation(msg.GetEvent().GetAction()), metrics.ComplianceOperatorInfo)
-	// Do something
-	return nil
+	if !features.ComplianceEnhancements.Enabled() {
+		return nil
+	}
+
+	var operatorErrors []string
+	operatorInfo := &storage.ComplianceIntegration{
+		Version:   msg.GetComplianceOperatorInfo().GetVersion(),
+		ClusterId: clusterID,
+		Namespace: msg.GetComplianceOperatorInfo().GetNamespace(),
```

Comment:
Consider augmenting this object with a more user-friendly cluster name.

Reply:
@md2119 I do return the cluster name in the API object.  We do not seem to store the cluster name in storage objects very often.  Would you like it stored in the storage object as well or is returning it to the user in the API version of the object what you are really getting after?

https://github.com/stackrox/stackrox/blob/a36951f95609a5349a84236bb88d4e293da7c92c/central/complianceoperator/v2/integration/service/convert.go#L26

Reply:
Okay, since the API object includes cluster name, it is okay to skip the name from storage for now.